src/ExchangeBase.ts

```diff
@@ -66,6 +68,62 @@ export abstract class ExchangeBase {
     }
   }
 
+  public async fetchDeposits(
+    address: string,
```

Comment:
you should use object input instead of positional argument

Reply:
ie: `{address, amountInSats}`

Reply:
done